php/commands/media.php

```diff
@@ -203,15 +207,15 @@ private function _make_copy( $path ) {
 		return $filename;
 	}
 
-	private function _process_regeneration( $id, $skip_delete = false ) {
+	private function _process_regeneration( $id, $skip_delete = false, $only_missing = false ) {
 		$image = get_post( $id );
 
 		$fullsizepath = get_attached_file( $image->ID );
 
-		$att_desc = sprintf( '"%1$s" (ID %2$d).', get_the_title( $image->ID ), $image->ID );
+		$att_desc = sprintf( '"%1$s" (ID %2$d)', get_the_title( $image->ID ), $image->ID );
 
 		if ( false === $fullsizepath || !file_exists( $fullsizepath ) ) {
-			WP_CLI::warning( "Can't find $att_desc" );
+			WP_CLI::warning( "Can't find $att_desc." );
```

Comment:
Ah, I see. It's not necessary to wrap with curly braces as it's a simple variable name. It would maybe make sense with object properties or multi level arrays. More examples at https://php.net/manual/en/language.types.string.php#language.types.string.parsing.complex.
Why are you after this? Is it causing you any issues or so?